docs/manual/general.rst

```diff
@@ -0,0 +1,217 @@
+4. Na het doorklikken op één item op de lijst zie je een detailweergave
+
+We gaan nu dieper in op wat je kan in lijst- en detailweergaves.
+
+.. _manual_general_list:
+
+Lijstweergave
+-------------
+
+Als voorbeeld zie je de lijstweergave van *Producten*:
+
+.. image:: assets/product_list.png
+    :width: 100%
+    :alt: Productenlijst
+
+1. De meeste lijstweergaves hebben een zoekveld waarmee je de lijst van
+   gegevens kan doorzoeken. Vaak zoeken deze op identificatie, UUID of een
+   ander karakteristiek attribuut.
+
+2. Aan de rechterzijde is er meestal een set aan filters beschikbaar. Deze
+   laten je toe om snel de resultaatset te reduceren. Filters kunnen
+   gecombineerd worden (combinaties werken als EN-filter).
+
+3. Kolommen zijn sorteerbaar - klik op het kolomhoofd om oplopend te sorteren.
+   Klik een tweede keer om aflopend te sorteren. Je kan sorteren op meerdere
+   kolommen - er verschijnt dan een nummer die aangeeft op welke kolommen er
+   in welke volgorde gesorteerd wordt.
+
+4. In de lijstweergave zijn *bulk acties* beschikbaar. Selecteer de objecten
+   waarop je de bulk actie wil toepassen door het vinkje links aan te vinken.
+   Kies vervolgens in de dropdown te actie die je uit wil voeren.
+
+   .. warning:: Merk op dat het verwijderen van objecten deze objecten ook echt
+      **permanent** verwijdert! Het is zelden nodig om objecten te verwijderen.
+
+5. Typisch is de eerste kolom in een lijstweergave een klikbare link. Door deze
+   aan te klikken ga je naar de :ref:`manual_general_detailview` van dat object.
+
+6. Rechtsboven heb je typisch een knop om nieuwe objecten toe te voegen. Deze
+   opent een formulier om de objectgegevens in te vullen.
+
+.. _manual_general_detailview:
+
+Detailweergave
+--------------
+
+In de detailweergave zie je de gegevens/attributen van één enkel object, al
+dan niet aangevuld met de gerelateerde objecten.
+
+Als voorbeeld zie je (een deel van) de detailweergave van een zaak:
+
+.. image:: assets/product_detail.png
+    :width: 100%
+    :alt: Product detail
+
+1. De attributen van de zaak worden opgelijst als bewerkbare velden. Sommige
+   attributen zullen niet bewerkbaar zijn, en als je geen bewerkrechten hebt
+   zie je alles als alleen-lezen. Verplichte velden worden in het vet gedrukt,
+   terwijl optionele velden normaal gedrukt zijn. Indien beschikbaar, dan wordt
+   onder het veld een extra helptekst getoond die meer context geeft over de
+   betekenis van een veld.
+
+2. Gerelateerde objecten worden vaak via een vergrootglas ingesteld. Wanneer je
+   het icoon aanklikt, dan wordt er een lijstscherm geladen waarin je het
+   gerelateerde object kan selecteren. Na selectie staat het database-ID
+   ingevuld in het veld.
+
+3. Gerelateerde objecten worden *inline* getoond. Zo zul je bij een zaak een
+   aantal inlines zien: statussen, zaakobjecten, zaakinformatieobjecten...
+   Dit zijn allemaal relaties aan deze specifieke zaak.
```

Comment:
- 2 needs to change since open producten does not use the magnifying glass like open zaak.
- 3 product currently does not have inlines, i think its fine to make it more generic without referencing product.